cmd/bundle/init.go

```diff
@@ -59,7 +59,7 @@ func newInitCommand() *cobra.Command {
 		} else {
 			return errors.New("please specify a template")
 
-			/* TODO: propose to use default-python (once template is ready)
+			/* TODO: propose to use default-python (once #708 is merged)
```

Comment:
#708 is now merged.

Reply:
👍 that's covered by https://github.com/databricks/cli/pull/725. I'll just click merge here to get things moving.